src/components/TextField/TextField.tsx

```diff
@@ -43,6 +43,7 @@ type InputMode =
   | 'text'
   | 'decimal'
   | 'numeric'
+  | 'numeric-no-spinner'
```

Comment:
This isn't a valid input type, unless it's super new and I don't know about it. I'm worried it will break accessibility. Can we add a prop to hide the spinner instead?

Reply:
Hey Kyle! Initially, I thought about doing a prop to hide the spinner, but was told that perhaps a new type would be better? https://shopify.slack.com/archives/CQK6FNLJK/p1602612476356400 I might've misunderstood what Ben meant, thank you :)